test/pod.persona.test.ts

```diff
@@ -0,0 +1,229 @@
+import { ethers } from 'ethers';
+import axios from 'axios';
+import { init, getPod } from '../src';
+import * as createSafe from '../src/lib/services/create-safe-transaction';
+import { userAddress2, constructGqlGetUsers, getSafeTransactionFixture } from './fixtures';
+import * as fetchers from '../src/fetchers';
+import * as utils from '../src/lib/utils';
+
+// Some of these tests run long.
+jest.setTimeout(7500);
+
+const expectedPod = {
+  admin: '0xf0C7d25c942264D6F21871c05d3dB3b98344b499',
+  id: 6,
+  name: '1-member-pods.pod.eth',
+  safe: '0xC99cE75a83B068DABF48c48f91C62a30fe0afBd2',
+  member: '0x61De0bbb6C8215Af3f821FE4884A28bc737f98D3', // random member
+};
+
+function mockGetPodFetchersByAddress(opts?: { overrideAdmin?: string }) {
+  const admin = opts?.overrideAdmin ? opts.overrideAdmin : expectedPod.admin;
+  jest.spyOn(fetchers, 'getPodFetchersByAddressOrEns').mockResolvedValueOnce({
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    Controller: { podAdmin: jest.fn().mockResolvedValueOnce(admin) },
+    Safe: {
+      address: expectedPod.safe,
+      nonce: jest.fn().mockResolvedValueOnce({
+        toNumber: jest.fn().mockImplementation(() => 5),
+      }),
+      getThreshold: jest.fn().mockResolvedValueOnce({
+        toNumber: jest.fn().mockImplementation(() => 10),
+      }),
+    },
+    podId: expectedPod.id,
+    Name: { name: expectedPod.name },
+  });
+}
+
+const provider = new ethers.providers.InfuraProvider('goerli', {
+  infura: '69ecf3b10bc24c6a972972666fe950c8',
```

Comment:
⚠️ Doesn't need to block this PR, but do you mind moving this to an env var for this repo as well? I see 10 instances of it: https://github.com/search?q=org%3Aorcaprotocol+69ecf3b10bc24c6a972972666fe950c8&type=code

Is this a development key or do we use it in prod as well?

@heyitschun Do you mind adding a low priority backlog ticket to eventually rotate this infura key? It's not the end of the world if it leaks, but it's good hygiene and practice for us to separate prod keys regardless.